### PR TITLE
GH Actions: auto update copyright year

### PR DESCRIPTION
**Pull request by MetRonnie**
Another one...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,18 @@
 External trigger functions for the Cylc workflow engine.
 
 <img src="https://travis-ci.org/cylc/cylc-xtriggers.svg?branch=master" alt="build:">
+
+## Copyright and Terms of Use
+
+Copyright (C) 2008-<span actions:bind='current-year'>2020</span> NIWA & British Crown (Met Office) & Contributors.
```
Comment (kinow) [inline README.md:8]: @MetRonnie should we use 2021 here? I think otherwise the happy-new-year action would run only 1st january, 2022?

**Review by kinow — APPROVED** [review]

**Comment by oliver-sanders** [comment]
This repo is currently abandoned, the code (with the exception of the kafka stuff) has been migrated back into Cylc Flow. The xtriggers here [shouldn't register](https://github.com/cylc/cylc-xtriggers/issues/12) yet anyway.

Will reboot this repo towards or after 8.0.0 and add an entry-point (or upgrade to whatever plugin system we pick) to get this working.

Note that the remaining example in this repo, kafka should be re-implemented as a main_loop plugin (which would be much more efficient and less laggy), but this requires the data store to settle down a bit first (need a task event stream we can tune into).

Not sure what the intention for this repo is going forwards? If this is a community repo should it share the Cylc GPL copyright or be more permissive? Moreover should it be more general than an xtrigger repo, more like cylc-collab or cylc-plugins?

File: README.md (hunk 1 above)
Comment (MetRonnie) [reply README.md:8]: (I think I've added a `workflow_dispatch` trigger so you could manually run it after it gets merged (if it does))

**Comment by MetRonnie** [comment]
Closing this as
> Scheduled workflows are disabled automatically after 60 days of repository inactivity.

anyway

**Pull request closed without merging**